Source/JavaScriptCore/create_hash_table

```diff
@@ -447,7 +447,7 @@ sub output() {
         my $hashTableString = "";
 
         if ($compactSize != 0) {
-            $hashTableString .= "static const struct CompactHashIndex ${nameIndex}\[$compactSize\] = {\n";
+            $hashTableString .= "constinit static const struct CompactHashIndex ${nameIndex}\[$compactSize\] = {\n";
```

Comment:
Typically, I think we put the linkage before the volatility. So to be consistent I think we should do `static continent const` for these.